### PR TITLE
Manual edits needed for 0.10.2 release

### PR DESCRIPTION
**Pull request by guozhangwang**

### CODE REVIEW

**Comment by guozhangwang** [comment]
@ewencp @derrickdoo for reviews.

**Comment by guozhangwang** [comment]
@ewencp Can I merge this patch now since 0.10.2 is released already? I will rebase the patch of course.

**Comment by ewencp** [comment]
@guozhangwang I had closed the vote, but release was still in the works (was doing some of this on a plane...). Once artifacts show up in maven and the artifacts have been released, this can be merged, although due to it changing some defaults to 0.10.2 we probably want to have it coincide with the release updates to the downloads page (i.e. the last website update steps for the release).

**Comment by guozhangwang** [comment]
Sounds good, I can add the part for the download page.

**Comment by guozhangwang** [comment]
@ewencp updated downloads page, also added a few kafka meetups.

We can merge it once https://archive.apache.org/dist/kafka/ has 0.10.2.0 shown up.

File: downloads.html [hunk 1]
```diff
@@ -4,329 +4,361 @@
...
-          </li>
-          <li>Scala 2.12 (pre-alpha) &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.10.1.1/kafka_2.12-0.10.1.1.tgz">kafka_2.12-0.10.1.1.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.12-0.10.1.1.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.10.1.1/kafka_2.12-0.10.1.1.tgz.md5">md5</a>)
-          </li>
+            <li>
+                <a href="https://archive.apache.org/dist/kafka/0.10.2.0/RELEASE_NOTES.html">Release Notes</a>
+            </li>
+            <li>
+                Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/0.10.1.1/kafka-0.10.2.0-src.tgz">kafka-0.10.2.0-src.tgz</a> (<a href="https://dist.apache.org/repos/dist/release/kafka/0.10.2.0/kafka-0.10.2.0-src.tgz.asc">asc</a>, <a href="https://dist.apache.org/repos/dist/release/kafka/0.10.2.0/kafka-0.10.2.0-src.tgz.md5">md5</a>)
```
Comment (ewencp) [inline downloads.html]: @guozhangwang Looks like the download link is slightly wrong -- it has 0.10.1.1 as directory and 0.10.2.0 in the link. I think it's just this one source download, the rest of the binary artifacts look ok.

**Review by ewencp — APPROVED** [review]
@guozhangwang Thanks for taking care of those final updates. One minor issue, but otherwise LGTM once the artifacts appear.

**Comment by guozhangwang** [comment]
Merged.

**Pull request closed without merging**